app/src/main/res/layout/dialog_buy_buttons.xml

```diff
@@ -25,12 +26,10 @@
 
   <Button
       android:id="@+id/buy_button"
-      style="@style/ButtonStyle.Alternative"
+      style="@style/ButtonStyle"
```

Comment:
The difference between this two layout is that the disable state for the alternative is grey and for the standard is invisible. Is this change intentional?